$OpenBSD: patch-gen_book_cc,v 1.1 2019/03/17 15:01:21 cwen Exp $
ports-gcc fix for:
gen_book.cc:519:29: error: 'strcmp' was not declared in this scope
Index: gen_book.cc
--- gen_book.cc.orig
+++ gen_book.cc
@@ -25,6 +25,8 @@
 
 #include <vector>
 
+#include <string.h>
+
 #include "book.h"
 #include "hash.h"
 #include "log_proc.h"
